docs/reference/pyproject-settings.rst

```diff
@@ -31,3 +31,13 @@ This page lists the configuration settings supported by ``meson-python`` in the
 
    * - ``tool.meson-python.args.install``
      - Extra arguments to be passed to the ``meson install`` command.
+
+Usage example:
```

Comment:
possible improvement:

I think this may be better as sub-section.

```suggestion
Example
=====
```

Reply:
This was on purpose - subsection heading look terrible in our Sphinx theme, and we should not over-use them.